react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,37 +1,25 @@
-import React, { Component } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 
-import Square from '../Square';
+import SquareList from '../SquareList';
+import { boardRows } from '../../constants';
 
-import styles from './styles.module.scss';
+function Board({ squares, onClick }) {
+  const [firstRow, secondRow, thirdRow] = boardRows;
 
-class Board extends Component {
-  renderSquare() {
-    return <Square />;
-  }
-
-  render() {
-    const status = 'Next player: X';
-    return (
-      <div>
-        <div className={styles.status}>{status}</div>
-        <div className={styles.boardRow}>
-          {this.renderSquare(0)}
-          {this.renderSquare(1)}
-          {this.renderSquare(2)}
-        </div>
-        <div className={styles.boardRow}>
-          {this.renderSquare(3)}
-          {this.renderSquare(4)}
-          {this.renderSquare(5)}
-        </div>
-        <div className={styles.boardRow}>
-          {this.renderSquare(6)}
-          {this.renderSquare(7)}
-          {this.renderSquare(8)}
-        </div>
-      </div>
-    );
-  }
+  return (
+    <fragment>
+      <SquareList boardRow={firstRow} squares={squares} onClick={onClick} />
+      <SquareList boardRow={secondRow} squares={squares} onClick={onClick} />
+      <SquareList boardRow={thirdRow} squares={squares} onClick={onClick} />
+    </fragment>
+  );
 }
 
+
+Board.propTypes = {
+  squares: PropTypes.arrayOf(PropTypes.string),
```

Comment:
Don't change it, I don't care so much and I don't want you to waste time for this, but I usually prefer doing the following:
```
import { arrayOf, func, string } from 'prop-types';

...
Board.propTypes = {
  squares: arrayOf(string);
  onClick: func
}
```